spec/takeaway_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'takeaway'
+
+describe Takeaway do
+  let(:menu_klass) { double(:menu_klass, new: menu) }
+  let(:order_klass) { double(:order_klass, new: order) }
+
+  let(:menu) { double(:menu, show: dishes, dishes: dishes)}
+  let(:dishes) { {menuitem1: 1, menuitem2: 1.50 } }
+  let(:order) { double(:order) }
+
+  subject(:takeaway) { described_class.new(order_klass: order_klass, menu_klass: menu_klass) }
```

Comment:
Metrics/LineLength: Line is too long. [94/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)